src/app/user-form/user-form.component.ts

```diff
@@ -1,15 +1,36 @@
 import { Component, OnInit } from '@angular/core';
+import {FormBuilder, FormGroup, Validators} from "@angular/forms";
+import * as historyActions from "../core/store/actions/history.actions";
+import {Store} from "@ngrx/store";
+import {IAppState} from "../core/store/state/app.state";
+import {selectUserDataLoading} from "../core/store/selectors/user-data.selectors";
+import {selectHistoryLoading} from "../core/store/selectors/history.selectors";
```

Comment:
could you to structure imports

first should be angulars imports and then our imports